lib/src/platform/src/platform_object.dart

```diff
@@ -3,12 +3,14 @@ import 'dart:async';
 import 'package:ably_flutter/ably_flutter.dart';
 import 'package:meta/meta.dart';
 
+/// @nodoc
 /// A representation of a Platform side instance (Android, iOS).
 abstract class PlatformObject {
   Future<int>? _handle;
   final Platform _platform = Platform();
   int? _handleValue; // Only for logging. Otherwise use _handle instead.
 
+  /// @nodoc
   /// immediately instantiates an object on platform side by calling
```

Comment:
All comments in this file could be "upgraded" to sentences - starting with a capital letter and ending with the dot if we want to

Reply:
Done in 19b0814c30acaf767beee3ed6eecd362b672f012